scripts/remove/lazylibrarian.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
In general, we don't do `echo_`s during removers, could just switch them to `echo_log_only`

Reply:
Sure. I'll put some TODOs in to change them to log_only near PR end. Would like the output to be sent to CLI for the time being.

Reply:
Pro tip: run `tail -f /root/logs/swizzin.log &` and hit enter. You'll get the full log including log only and forwarded output into your terminal as the script runs.